From 9da41b60bb621502bd1dfff8b9ab6bf2fbde20f2 Mon Sep 17 00:00:00 2001
From: kkapranov <45823870+kkapranov@users.noreply.github.com>
Date: Thu, 7 Jan 2021 17:51:32 +0200
Subject: [PATCH 2/2] Create README.md

---
 README.md | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/README.md b/README.md
index ddb5683..063b596 100644
--- a/README.md
+++ b/README.md
@@ -1 +1,4 @@
-# imx8mm-a10
\ No newline at end of file
+
+# imx8mm-a10
+Android q10 for imx8mm 
+VERY EXPERIMENTAL!
-- 
2.11.0

